File: ProgramingBasics/SimpleCalculations/05.TrapeziodArea/src/TrapeziodArea.java
import java.util.Scanner;

public class TrapeziodArea {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double h = scanner.nextDouble();

        double area = (a + b) * h / 2;
        System.out.println(area);
    }
}

File: ProgramingBasics/SimpleCalculations/05.TrapeziodArea/05.TrapeziodArea/TrapeziodArea.cs
using System;

namespace _05.TrapeziodArea
{
    public class TrapeziodArea
    {
        public static void Main(string[] args)
        {
            var a = double.Parse(Console.ReadLine());
            var b = double.Parse(Console.ReadLine());
            var h = double.Parse(Console.ReadLine());
            var area = (a + b) * h / 2;

            Console.WriteLine(area);
        }
    }
}
